x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/recurrence_scheduler.tsx

```diff
@@ -0,0 +1,750 @@
+/*
```

Comment:
General -> 
I think I am more in favor to create an handler around the setter so I can have a callback to call the onChange instead of using `useMemo` and `useEffect`

Reply:
Tracking here: https://github.com/elastic/kibana/issues/134093